airlock/templates/file_browser/file.html

```diff
@@ -43,15 +51,26 @@
           </form>
         {% endif %}
       {% elif is_output_checker %}
-        <div>
-          {% if not file_reset_review_url %}
-            <span class="file-status file-status--warning">No status</span>
-          {% elif file_approve_url %}
-            <span class="file-status file-status--danger">Changes requested</span>
-          {% elif file_reject_url %}
-            <span class="file-status file-status--success">Approved</span>
-          {% endif %}
-        </div>
+        {% if not path_item.is_supporting and not path_item.is_withdrawn %}
```

Comment:
This `if` shouldn't be needed; the file approve/reset/reject urls are all None if the filetype is supporting or withdrawn
https://github.com/opensafely-core/airlock/blob/85083f408980679472759a0bad397971ad044a73/airlock/views/request.py#L210

Reply:
When I remove that if statement, I get the "no status" showing:

![](https://github.com/opensafely-core/airlock/assets/24863179/9b9161bf-abd7-4432-ac51-ff1eb842ae94)

Reply:
Lovely, that's worked.